bedrock/firefox/templates/firefox/whatsnew/fx54/whatsnew-54.html

```diff
@@ -12,6 +12,12 @@
 
 {% extends "firefox/base-resp.html" %}
 
+{% block experiments %}
+  {% if switch('firefox-experiment-whatsnew-55', variation_locales) %}
```

Comment:
There are more things like this, such as images etc... but I'm guessing it's all as expected?